websites/F/FRAnime/presence.ts

```diff
@@ -0,0 +1,132 @@
+// Init the client
+const presence = new Presence({
+		clientId: "1102272266278027296",
+	}),
+	// Define playerAssets
+	playerAssets = {
+		logo: "https://i.imgur.com/Fe6SuHW.png",
+		logoBg: "https://i.imgur.com/byFtXWd.png",
+		pause: "https://i.imgur.com/0A75vqT.png",
+		play: "https://i.imgur.com/Dj5dekr.png",
+		search: "https://i.imgur.com/C3CetGw.png",
+	},
+	// Define the player selector, that will be used to detect the correct player
```

Comment:
Comments like these and others throughout this presence are redundant and should be removed